subprojects/build-cache/src/integTest/groovy/org/gradle/caching/internal/BuildCacheBuildOperationsIntegrationTest.groovy

```diff
@@ -391,7 +387,7 @@ class BuildCacheBuildOperationsIntegrationTest extends AbstractIntegrationSpec {
 
         packOp.details.cacheKey == remoteMissLoadOp.details.cacheKey
         def localCacheArtifact = localCache.cacheArtifact(packOp.details.cacheKey.toString())
-        !localCacheArtifact.exists()
+        localCacheArtifact.exists()
```

Comment:
I don't think this test should be called `does not emit operations for custom local cache implementations`. Can this test be removed maybe? Is the thing already covered in the other tests in this class (it should be).

Reply:
Good point, removed.